src/main/java/seedu/address/model/order/stage/SentForDeliveryState.java

```diff
@@ -0,0 +1,18 @@
+package seedu.address.model.order.stage;
+
+public class SentForDeliveryState implements StageState {
```

Comment:
would sentfordeliverystate be too troublesome? maybe just under preparation, ready for delivery and received is enough?